src/main/java/se/sundsvall/precheck/integration/partyassets/PartyAssetsProperties.java

```diff
@@ -0,0 +1,13 @@
+package se.sundsvall.precheck.integration.partyassets;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import se.sundsvall.precheck.integration.AbstractIntegrationProperties;
+
+@Getter
+@Setter
+@ConfigurationProperties(prefix = "integration.party-assets")
+class PartyAssetsProperties extends AbstractIntegrationProperties {
+    private OAuth2 oAuth2 = new OAuth2();
```

Comment:
What is this?

Reply:
It is a class defined in ``AbstractIntegrationProperties`` on line 26, representing all the authenticating properties like ``tokenUrl``, ``ClientId``, and ``ClientSecret`` used by the integrations.

It has been taken from other repositories by Sundsvall-kommun, primarily [``api-service-sms-sender``](https://github.com/Sundsvallskommun/api-service-sms-sender/blob/ba69d4d9b0fff4d1e81c00b26ae251097010c18e/src/main/java/se/sundsvall/smssender/provider/SmsProviderProperties.java#L31) and [``api-service-case-status``](https://github.com/Sundsvallskommun/api-service-case-status/blob/71a19d95006f780d6805b03e7dacad8516723d9e/src/main/java/se/sundsvall/casestatus/integration/AbstractIntegrationProperties.java#L26).

Should it not be used in this context?